experiments/kompanion/cmd/export/export.go

```diff
@@ -141,66 +248,60 @@ func run(_ *cobra.Command, _ []string) {
 	sort.Slice(resources, func(i, j int) bool {
 		return resources[i].SingularName < resources[j].SingularName
 	})
+
+	// todo acpana debug logs
 	// log.Printf("Going to iterate over the following resources %+v", resourcesToName(resources))
 
 	namespaces, err := clientset.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
 	if err != nil {
 		log.Fatalf("Error fetching namespaces: %s\n", err)
 	}
 
-	reportName := timestampedName("report")
+	reportName = timestampedName("report")
 	if err := os.Mkdir(filepath.Join(reportName), os.ModePerm); err != nil {
 		log.Fatalf("Could not create \"report\" directory: %w", err)
```

Comment:
Nit: I always like to return the error, rather than panic-ing.  Cobra has a RunE method you can implement instead of Run